crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -38,9 +38,13 @@ pub(crate) struct TransactionBlock {
     pub native: NativeSenderSignedData,
 }
 
+/// The kind of transaction block: a system transaction or a programmable transaction.
```

Comment:
I don't see a relevant annotation on this type -- does this doc comment need to be moved somewhere?

Reply:
Yes!!!